src/ensembl/src/server/index.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import server from './server';
+
+const port = 8080;
+
+server.listen(port, () => {
+  // eslint-disable-next-line
+  console.log(`Example app listening at http://localhost:${port}`);
```

Comment:
This is not an example app anymore right? :D

Reply:
Replaced with a more appropriate message.